README.md

```diff
@@ -63,6 +63,19 @@ opus list .3
 opus list 1
 opus ls 1
 opus l 1
+# sort list output by task property
+opus list --sort_by=id
```

Comment:
Personally, I prefer using dashes instead of underscores for separating words in arguments, but go with what every you want.
Rest look good to me :)

Reply:
youre right, looks a lot cleaner, will change